docs/configuration/configuration.md

```diff
@@ -95,6 +95,18 @@ This option does exactly the same as `validation.ignoreParams`, but provides pip
 validation.defaultIgnoreParams = ["param1", "param2"] // default: []
 ```
 
+## maxValueLength
```

Comment:
I find the name a bit confusing, what do you think about `maxErrorValueLength`? too long?

Reply:
Yeah that's a lot clearer but is getting very long too :thinking:

I agree that the name could be better, I'll see if I can find a better, shorter one otherwise I'll just go with yours

Reply:
how about `maxErrValSize`?

Reply:
sounds good 👍